juicy-jsoneditor.html

```diff
@@ -41,10 +36,9 @@
                     name: { type: String, observer: "nameChanged" },
                     search: { type: Boolean, observer: "searchChanged", value: true },
                     indentation: { type: String },
-                    history: { type: Boolean, value: false }
+                    history: { type: Boolean, value: false },
+                    editor: {type: Object},
                 },
-                editor: null,
-                observer: null,
```

Comment:
We still use the observer, I'd keep it in the prototype, to make the API known beforehand.

Reply:
There was a reference to editor being undefined, and the move to polymer properties resolved that. The removal of observer was a misstep. I'll re-add and recheck. Thanks for the feedback!